packages/wire-format/src/types.ts

```diff
@@ -140,8 +140,9 @@ export const isVirtuallyFund = guard<VirtuallyFund>('VirtuallyFund');
 export const isFundGuarantor = guard<FundGuarantor>('FundGuarantor');
 
 // channel requests
-type GetChannel = {type: 'GetChannel'; channelId: Bytes32};
-export type ChannelRequest = GetChannel;
+type GetChannel = {type: 'GetChannel'; channelId: string};
+type ProposeLedger = {type: 'ProposeLedger'; channelId: string; outcome: Outcome};
+export type ChannelRequest = GetChannel | ProposeLedger;
```

Comment:
As discussed yesterday, moving to a two-phase commit scheme for ledger updates is a change with significant implications, and deserves a proper proposal and discussion. I remember you saying that the change wasn't necessary for ledger channels to work. Is it possible to remove the changes from this PR?

Reply:
The written-before-merge version of the code was also a two-phase commit, this change just:
- moved the storage of the commits from being in the Channels table as _signed states_ to being in there as two new columns
- moved the communication of the commits from `signedStates` to `requests`
Do you still think we should go back?

Reply:
The difference is quite large in the case where the peers agree on what ledger update to make, where one version requires an extra round of messaging. Since we expect to be in that world in the medium term, it seems like a sensible default to avoid the extra messaging.

Reply:
#2761